src/assets/css/style.css

```
@@ -181,6 +188,35 @@ button {
   font-weight: 500;
 }
 
+.empty__board {
+  margin: auto;
+  text-align: center;
+  padding-top: 8.125rem;
+}
+
+.empty__board-title {
+  font-weight: 700;
+  font-size: 1.5rem;
+  line-height: 1.75rem;
+  color: var(--black);
+  padding-top: 2rem;
+}
+
+.todo__pagination {
+  padding-top: 3.563rem;
+  text-align: center;
+}
+
+.todo__pagination-btn {
```

without writing  it again, you could do this
```suggestion
.todo__pagination {
  padding-top: 3.563rem;
  text-align: center;
  
  &-btn {
  }
}

```

---

Implemented. Thanks Bhai!